webapp/src/pages/Recipe/Recipe.tsx

```diff
@@ -1,10 +1,82 @@
 import React from 'react';
-import './Recipe.css';
+import { useParams } from 'react-router-dom';
+import { Panel } from 'primereact/panel';
+import { Splitter, SplitterPanel } from 'primereact/splitter';
+// ? Not sure why this library can't be found even though when I console.log(uuidv4()) it works?
+// import { v4 as uuidv4 } from 'uuid';
+import PlaceHolder from '../../assets/images/samosa.jpg';
+import { useQuery } from 'urql';
+import './Recipe.scss';
+// * homemade uuid ^_^ since the libaray won't work!
+const uuid: number = Math.floor(Math.random() * 10000);
 
 export default function Recipe() {
+  //* retrive recipe ID from url path
+  const { ID } = useParams();
```

Comment:
Not for this sprint, but this is the kind of info I'd grab in a page and then pass into a component within the page, to help with reusability and why I'd split it out